Makefile

```diff
@@ -16,12 +29,19 @@ ci.integ: ci.test
 fuzz:
 	go test -timeout=300s ./fuzzy
 
-deps:
+deps: dep-linter
 	go get -t -d -v ./...
 	echo $(DEPS) | xargs -n1 go get -d
 
+lint:
+	gofmt -s -w .
+	golangci-lint run -c .golangci-lint.yml $(FMT) .
+
+dep-linter:
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(ENV)/bin v1.16.0
```

Comment:
I am always a little hesitant to pipe from curl to sh in that way. Very easy to get screwed that way. Especially since this would be done for every `deps` run.

Reply:
@i0rek Would you prefer I curl a known release on GH and checksum it? Sorry, I've never scripted package management the right way before, I don't know what I'm doing.

Reply:
Would just `go get`ting github.com/golangci/golangci-lint/cmd/golangci-lint work?

Reply:
Just took a look at their docs and what you have is their recommendation, I am fine with it. It might be easier to bubble the version up to the top of the Makefile for visibility and I would suggest decoupling `dep-linter` from `deps`. `Deps` is more for go dependencies whereas dep-linter is just installing this binary so it may be semi confusing if I just want to build and may never want to run `make lint` to have a binary pulled all the time.